SpatialGDK/Source/SpatialGDKFunctionalTests/Public/SpatialFunctionalTest.h

```diff
@@ -430,25 +423,44 @@ class SPATIALGDKFUNCTIONALTESTS_API ASpatialFunctionalTest : public AFunctionalT
 
 	FString GetMapName() override;
 
+	// Logs messages originating from Require statements used in a test.
+	void LogRequireMessages(const FString& Message, bool bPassed);
+
+	/*
+	 * Adds support for multiple processes testing by ensuring that when starting a test, the Editor process, which is a Client, 
+	 * will send a Server RPC, by calling SpatialFunctionalTestFlowController::ServerNotifyRunTest, that will then call
+	 * AFunctionalTest::CrossServerRunTest, following the flow of starting a test when using a single process.
+	 */
+	virtual void CallRunTest(const TArray<FString>& Params = TArray<FString>()) override;
+
+	/*
+	 * Adds support for multiple processes testing by propagating logs originating from Require statements to the Editor Client process.
+	 * Having a MultiCast allows the Editor process to correctly report a tests pass/failure.
+	 * Its implementation simply calls ASpatialFunctionalTest::LogRequireMessages.
+	*/
+	UFUNCTION(NetMulticast, Reliable)
+	void MulticastLogRequireMessages(const FString& Message, bool bPassed);
```

Comment:
hmmmmmmmmmmmm
If I understand correctly, you are propagating requires as a special case, right? So only specific logs get propagated to the editor process?

Reply:
That is correct

Reply:
Closed as we unfortunately don't have the time to change this, will be added as a current limitation and a follow-up ticket will be created for this.